foosball/games/models.py

```diff
@@ -0,0 +1,56 @@
+import datetime
+
+from django.core.validators import MaxValueValidator
+from django.db import models
+from django.utils.translation import ugettext as _
+from model_utils.models import TimeStampedModel
+
+from foosball.users.models import User
+
+
+class Game(TimeStampedModel):
+    created_by = models.ForeignKey(
+        User, related_name="games_created", editable=False, null=True, verbose_name=_("created by")
+    )
+    modified_by = models.ForeignKey(
+        User, related_name="games_modified", editable=False, null=True, verbose_name=_("modified by")
+    )
+
+    played_at = models.DateTimeField(
+        default=datetime.datetime.now,
+        verbose_name=_("played at"),
+        help_text=_("Time when the game was played")
+    )
+
+    class Meta:
+        ordering = ['-played_at']
+
+    def __str__(self):
+        return " - ".join(map(str, self.teams.all()))
+
+
+class GameTeam(models.Model):
```

Comment:
I feel prefixing models with `Game` is really not necessary, with a scoring app we're going to have only teams related to games, not any other teams? Especially as this is already namespaced in an app called `games` so they're always going to be *game teams*.